src/quo2/components/banners/banner/__tests__/banner_component_spec.cljs

```diff
@@ -15,3 +15,13 @@
                       (.toBeTruthy))
                   (-> (js/expect (rtl/screen.getByText "5"))
                       (.toBeTruthy))))
+
+(js/global.test "banner component fires an event when pressed"
+                (let [mock-fn (js/jest.fn)]
```

Comment:
The `mock-fn` binding should be declared inside the anonymous function. This is currently a mistake in other files too, which I believe you based your tests on.

In fact, in general, the entire body of the test should be put inside the anonymous function. This is quite important so that the test runner (Jest) can reliably run all setup/teardown callbacks.

Reply:
yeah looks like you are correct, basically I need to wrap everything here in a `fn` right?

```
(js/global.test "banner component fires an event when pressed"
           (fn [] (let [mock-fn (js/jest.fn)] ...
```

Reply:
Precisely, in Javascript we always define the test inside the anonymous function, i.e. we never define a variable *around* the anonymous function. Sure we can define it below describe and use before/after hooks, but that's another thing entirely.

```javascript
describe('some description', () => {
  it('test description', () => {
      // mocks, etc
  })
})
```

Reply:
yep, I'm familiar with this. Probably just an oversight with me being new to converting js -> cljs. 

I'll update the test files in a separate pr 👍